CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and includes an additional section for migration notes.
 - *ORCA-245* Updated resource policies related to KMS keys to provide better security.
 - *ORCA-318* Updated post_to_catalog lambda to match new Cumulus schema changes.
 - *ORCA-317* Updated the db_deploy task, unit tests, manual tests, research pages and SQL to reflect new inventory layout to better align with Cumulus.
+- *ORCA-249* Changed `mutipart_chunksize_mb` in lambda configs to `s3MultipartChunksizeMb`.
 
 ### Migration Notes
 
```

Comment:
I would add a bullet here for the variable name change to let others know how to adjust their step function for copy_to_glacier. I provided an example of what I am thinking below:
```
- Update your step function that includes the `copy_to_glacier` lambda with the new [configuration schema](address to config schema here) and [inputs schema](address to input schema here). 
    - The following config items have changed their names:
        -  `mutipart_chunksize_mb` is now `s3MultipartChunksizeMb`
        -  others we may have changed since 3.0.2
    - The following config items have been added:
        - providerId - Required and pulled from the CMA metadata
        - other ones we have added since 3.0.2
    - The following config items have been removed:
        - etc.....
    - The following input items have been modified
        -   etc ...  
```

Reply:
Added in the multipart change, but held off on others since provider_id isn't a new requirement.

Reply:
Actually they are as we didn't have any of those in 3.0.2. Correct? In your comment, you may also want to add that the collection configuration change that the key `mutipart_chunksize_mb` is now `s3MultipartChunksizeMb`. Remember the migration notes are for Operators and Developers deploying Cumulus.

Reply:
Ah, gotcha. Went with a copy-paste friendly approach.